src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -96,6 +96,7 @@ private void testGeneralQuestionOperations() throws Exception {
 
         testEditQuestionLink();
         testEditQuestionAction();
+        testEditVisibilityAction();
```

Comment:
There are several tests that edit the visibility and they verify the main html file.